### DISCUSSION TITLE
Audio sync off

### DISCUSSION THREAD

**Discussion by bilak** [croneter/PlexKodiConnect#2128] [Q&A]
Hello, I've installed kodi and this plugin on my lgtv. However it seems like to audio offset is different. Is there any easy way (without manually adjusting sound) to fix it? 
If I play the video through plex on the tv, audio is fine.
Thanks in advance

**Comment by brainzor** [comment]
For tv's I think using the app is best. If you really enjoy kodi, you can use this addon: https://forum.kodi.tv/showthread.php?tid=379179
I think kodi experience is better and more stable on an android device (Ugoos, shield,...). Goodluck
